tests/unit/LaravelPathsConfigurationTest.php

```diff
@@ -14,18 +14,29 @@ public function testLaravelPathsConfiguration(): void
         $config = new Config(paths: $paths);
 
         $this->assertSame(
-            ["app", "config", "database", "resources/lang", "routes", "tests"],
+            ["app", "bootstrap/app.php", "config", "database", "lang", "public/index.php", "routes", "tests"],
+            $config->options()["paths"],
+        );
+    }
+
+    public function testPreviousLaravelVersionPathsConfiguration(): void
```

Comment:
Maybe we will tell which version is it? When `Laravel` has version `10`, will this test need to be updated?

Reply:
Yes, but this will be a breaking change.